recipe/meta.yaml

```diff
@@ -10,32 +10,37 @@ source:
   sha256: 694a8e44c87657c59292ede72891eb91d34131f6531463aab3009191c77364a8
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
+    - flit-core 3.8.0
     - pip
-    - python >=3.7
-    - flit-core
```

Comment:
is this not needed anymore ? https://github.com/rspeer/ordered-set/blob/d921651b2737f5cfe711868fab558a8ef79e26ca/pyproject.toml#L2

Reply:
It's here https://github.com/AnacondaRecipes/ordered-set-feedstock/pull/2/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR19